### ISSUE TITLE
get started: run.py ?

### ISSUE THREAD

**Issue by ozett** [Casterke/hikvision-alertsystem#2]
is your code working? i got errors, when i "run.py" to get startet.
what are basic steps? cmd-line arguments needed?

thankx,
t

**Comment by Casterke** [comment]
> is your code working? i got errors, when i "run.py" to get startet.
> what are basic steps? cmd-line arguments needed?
> 
> thankx,
> t

Currently it's not maintained but I'm want to refactor the while code soon. Currently it's built to run inside a docker and that's the preferred way but you can try to run it locally by the run.py but it's needs the volumes/folders/configs.

**Comment by ozett** [comment]
yeah, i tried the code but gradually changed some bits to get it running.
but it kept me going the first step...looking forward to your refactoring!
👍

**Comment by ozett** [comment]
please add some lines on the readme.md to get started,
also if you dockerize it.

a simple example could help to ease the beginning. 
thx.
